packages/@ngtools/webpack/src/paths-plugin.ts

```diff
@@ -95,8 +95,12 @@ export class PathsPlugin implements Tapable {
         if (onlyModule) {
           aliasPattern = new RegExp(`^${excapedAlias}$`);
         } else {
-          let withStarCapturing = excapedAlias.replace('\\*', '(.*)');
-          aliasPattern = new RegExp(`^${withStarCapturing}`);
+          if (alias !== '*') {
+            let withStarCapturing = excapedAlias.replace('\\*', '(.*)');
+            aliasPattern = new RegExp(`^${withStarCapturing}$`);
+          } else {
+            aliasPattern = new RegExp(`^${excapedAlias}$`);
```

Comment:
So the pattern now is `^\\*$` which only matches a star? That seems really wrong.